addOns/ascanrulesBeta/src/test/java/org/zaproxy/zap/extension/ascanrulesBeta/Log4ShellScanRuleUnitTest.java

```diff
@@ -113,6 +116,32 @@ void shouldStopSendingPayloadsOnNonIoException() throws Exception {
         assertThat(httpMessagesSent, hasSize(1));
     }
 
+    @Test
+    void shouldReturnExpectedNumberOfAlertTags() {
+        // Given / When
+        Map<String, String> alertTags = rule.getAlertTags();
+        // Then
+        assertThat(alertTags.size(), is(equalTo(4)));
+    }
+
+    @Test
+    void shouldReturnExpectedExampleAlerts() {
+        // Given / When
+        List<Alert> alerts = rule.getExampleAlerts();
+        Alert alert1 = alerts.get(0);
+        Alert alert2 = alerts.get(1);
+        // Then
+        assertThat(alerts.size(), is(equalTo(2)));
+        assertThat(alert1.getAlertRef(), is(equalTo("40043-1")));
+        assertThat(alert1.getTags().size(), is(equalTo(5)));
```

Comment:
Should I comment here that it's general +1 because of the alerts being distinct? Or Should I add them both to the general case and remove one from each alert?

Reply:
IMO the `getAlertTags()` should return both CVEs and each alert their own.

Reply:
Hmm there doesn't seem to be a good way to have a map and remove entries.
Should I declare three constant maps?

Reply:
For the alerts it can check the presence and absence of the CVEs and for the scan rule tags check both are present.

Reply:
No sorry I meant actually within the rule

Reply:
Map.remove seemed to be ending up with a blank entry instead of actually removing in my testing. Tried it a few different ways.